### ISSUE TITLE
can't fill simple shape

### ISSUE THREAD

**Issue by hvbpb** [inkstitch/inkstitch#571]
Hi there,

I'm struggling to fill in a background shape. Filling works great on other objects in the same svg, just whatever I do to get a simple black background fill wont work!? Tried following the usage workflow. Even that doesn't work. 

The error I get is
Exception AttributeError: "'NoneType' object has no attribute 'GEOSSTRtree_destroy'" in <bound method STRtree.__del__ of <shapely.strtree.STRtree instance at 0x0CD148A0>> ignored

I'm finding that really odd. As other objects in the file work just fine...

edit: I've just run the file through a simulation and the background was simulated.. Is it going to embroider or not?!

**Comment by lexelby** [comment]
Wow, that is a _really_ weird error.  If you copy/paste that fill shape into a new document, does it still give that error?  If so, would you be willing to upload that file here so I can try out this fill object?  I'd like to see if I can get that error on my computer too.

**Comment by hvbpb** [comment]
I've checked the xml in the svg in the meanwhile and the embroidery commands are there. That would explain why the simulation is working I guess. Also, I suspect the background is in fact going to be embroidered if I try it in on a machine...
Copying and Pasting the same object to another file did provoke the error first.. When I resized it slightly, the error was gone! However, resizing did not do the trick on the original file. Also, I believe we're working with vectors and "Size" shouldn't matter.... 

[buggy_black_circle.zip](https://github.com/inkstitch/inkstitch/files/3828186/buggy_black_circle.zip)

**Comment by hvbpb** [comment]
I've checked the xml in the svg in the meanwhile and the embroidery commands are there. That would explain why the simulation is working I guess. Also, I suspect the background is in fact going to be embroidered if I try it in on a machine...
Copying and Pasting the same object to another file did provoke the error first.. When I resized it slightly, the error was gone! However, resizing did not do the trick on the original file. Also, I believe we're working with vectors and "Size" shouldn't matter.... 

[buggy_black_circle.zip](https://github.com/inkstitch/inkstitch/files/3828186/buggy_black_circle.zip)
